dataset_description.json

```diff
@@ -1,5 +1,17 @@
 {
 	"Name": "data_axondeepseg_sem",
-	"BIDSVersion":"1.6.0 - BEP031 v0.0.4",
+	"BIDSVersion": "1.6.0 - BEP031 v0.0.5",
+	"Authors": ["Pierre-Louis Antonsanti",
+	            "Marie-Hélène Bourget",
+	            "Julien Cohen-Adad",
+	            "Tanguy Duval",
+	            "Alexandru Foias",
+	            "Victor Herman",
+	            "Nafisa Husein",
+	            "Harris Nami",
+	            "Christian S. Perone",
+	            "Ariane Saliani",
+	            "Maxime Wabartha",
+	            "Aldo Zaimi"],
 	"License": "MIT"
```

Comment:
do we want to change it to CC0?

Reply:
I did not change it because the MIT License seems to be included in all of the Neuropoly repos.

We changed for CC0 only for the dataset that will be on the `bids-examples` repo.